com.unity.mobile.notifications/Runtime/Android/AndroidNotificationCenter.cs

```diff
@@ -49,6 +52,9 @@ public class AndroidNotificationCenter
         private static AndroidJavaObject s_CurrentActivity;
         private static bool s_Initialized;
 
+        /// <summary>
+        /// Initialize the Android notification center class. (Does this need to be public?)
```

Comment:
This can be internal. But sadly I don't think we can change it to internal at this point, as any API change should be at least in the minor version release...

We definitely should change this to internal in the future.